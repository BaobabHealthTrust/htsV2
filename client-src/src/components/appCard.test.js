import React from 'react';
import ReactDOM from 'react-dom';
import AppCard from './appCard';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: AppCard', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <AppCard />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})