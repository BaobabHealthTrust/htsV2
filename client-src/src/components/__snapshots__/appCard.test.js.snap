// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: AppCard should match its empty snapshot 1`] = `
<table
  border="0"
  cellPadding="5"
  style={
    Object {
      "borderCollapse": "collapse",
      "margin": "auto",
    }
  }
>
  <tbody>
    <tr>
      <td
        style={
          Object {
            "textAlign": "right",
            "width": "40px",
          }
        }
      >
        <img
          alt=""
          height="110"
          src={undefined}
        />
      </td>
      <td
        style={
          Object {
            "color": "#345db5",
            "fontSize": "30px",
          }
        }
      />
    </tr>
  </tbody>
</table>
`;
